cpp/ql/src/semmle/code/cpp/models/implementations/Swap.qll

```diff
@@ -2,7 +2,10 @@ import semmle.code.cpp.models.interfaces.DataFlow
 import semmle.code.cpp.models.interfaces.Taint
 
 /**
- * The standard function `swap`.
+ * The standard function `swap`. A use of `swap` looks like this:
+ * ```
+ * std::swap(obj1, obj2)
+ * ```
  */
 private class Swap extends DataFlowFunction {
```

Comment:
We might as well implement `AliasFunction` while we're at it, right? (Doesn't have to be in this PR, of course)

Reply:
Done.